3 Domain layer/CandidatesEvaluator.Contract/Queries/Question/GetQuestion.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace CandidateEvaluator.Contract.Queries.Question
+{
+    public class GetQuestion : IQuery<Models.Question>
+    {
+        public Guid OwnerId { get; set; }
+
+        public Guid CategoryId { get; set; }
+
```

Comment:
IMHO new lines between properties are redundant and other classes don't have that style.

Reply:
Done